### ISSUE TITLE
[BUG] Rails 7.0 won't work with current encryption setup 

### ISSUE THREAD

**Issue by etagwerker** [fastruby/dash#95]
**Before we start...:**

- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed
- [x] I'm reporting the issue to the correct repository (for multi-repository projects)

**Branch/Commit:**

`main` branch. 

**Expected behavior:**

The test suite should pass with both Rails 6.1 and 7.0 (edge) 

**Actual behavior:**

The test suite is not passing with Rails 7.0 due to an encryption configuration error 

**Steps to reproduce:**

It should be easy to reproduce if you have the latest version of rails (edge) 

1. Step 1: `next bundle update`
2. Step 2: `next bundle exec rspec`

**Context and environment:**

- Encryption is now natively supported in Rails 7.0 and `lockbox` is no longer needed: https://edgeguides.rubyonrails.org/active_record_encryption.html
- https://bigbinary.com/blog/rails-7-adds-encryption-to-active-record
- PR that introduced encryption and new configuration validations in Rails edge: https://github.com/rails/rails/pull/41659

**Logs**

Console log here: https://gist.github.com/etagwerker/9c37c3b0e0793fab992fac0363b76f05
Circle CI failure with Rails 7.0: https://app.circleci.com/pipelines/github/fastruby/dash/160/workflows/efcc7724-1964-4b65-a3fb-368729e38d4b/jobs/207

**Potential Solutions?**

- Migrate from `lockbox` to ActiveRecord encryption? 
- Do not use or load anything related to ActiveRecord encryption? 

**I will abide by the [code of conduct] (https://github.com/rubygems/rubygems/blob/master/CODE_OF_CONDUCT.md)**

**Comment by arielj** [comment]
I missed this issue and created a duplicated one https://github.com/fastruby/dash/issues/105, adding a reference here
